products/cpdb/python/admin_boundaries.py

```diff
@@ -0,0 +1,101 @@
+from dataclasses import dataclass
+from abc import abstractmethod
+
+
+@dataclass
+class AdminBoundary:
+    boundary_type: str
+    cpdb_boundary_type: str
+    table_name: str
+    geography_code: str
+    geography_name: str
+
+
+@dataclass
+class AdminBoundaries:
+    @abstractmethod
+    def generate_boundary(self, geography_number: int) -> AdminBoundary:
+        raise NotImplementedError("AdminBoundaries is an abstract class")
+
+
+@dataclass
+class CityCouncilDistricts(AdminBoundaries):
+    count: int
+    boundary_type: str = "City Council District"
+    cpdb_boundary_type: str = "council"
+
+    def generate_boundary(self, geography_number: int) -> AdminBoundary:
+        table_name = "city_council_district_" + str(geography_number)
+        geography_code = str(geography_number)
```

Comment:
can we call this geoid instead maybe?

Also, CCDs are supposed to be 2 digit i believe - would be good to lpad to two digits with zeroes

Reply:
they aren't 2 digits in the relevant table `cpdb_adminbounds`

<img width="520" alt="Screenshot 2024-05-08 at 2 50 19 PM" src="https://github.com/NYCPlanning/data-engineering/assets/7444289/5057732d-c4f3-4886-8723-25743b5dbde7">


and this value is used in
```sql
SELECT * FROM cpdb_adminbounds
WHERE
    cpdb_adminbounds.admin_boundary_type = '{{ boundary_type }}'
    AND cpdb_adminbounds.admin_boundary_id = '{{ geography_code }}'
```

Reply:
gonna go with `bounadry_id` to align with the column in `cpdb_adminbounds`